client/commands/run.js

```diff
@@ -4,7 +4,7 @@ const path = require("path");
 const { spawn } = require("../process");
 const { getConfig, replaceFileExtension } = require("../util");
 
-module.exports = function run({ outputFile, outputDir, debug }) {
+module.exports = function run({ outputFile, outputDir, debug, defaultOutputFileName }) {
```

Comment:
I get a bit confused by the naming of this. I think it might be better to send in the `fileToCompile` as it is instead. And just replace the extension to .vs here in the run command. What you you think?